### PR TITLE
Sort resources and schemas in OpenApi docs

### LINKED ISSUE #3996: Sorting of the resources in the (OpenApi) docs page
**API Platform version(s) affected**: 2.6.1

**Description**  
For the OpenApi docs, the resources are no more sorted as before.

**How to reproduce**  
Just download and run the application, create a few api resources and then navigate to the (OpenApi) docs page.

**Possible Solution**  
I have created a pull request at [#3995](https://github.com/api-platform/core/pull/3995) but for some reason the PhpStan check fails. The failure message comes from some ElasticSearch component which does not relate with my commits.

**Additional Context**  
If anyone wants to look into the PhpStan failure message, please feel free.

### PR DESCRIPTION
**Pull request by stiffroy**
| Q             | A
| ------------- | ---
| Bug fix?      | yes 
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no
| Tickets       | fixes #3996  
| License       | MIT
| Doc PR        | 

This is a very minor bug fix. With the introduction of the OpenApi models in version 2.6, the resources and the schemas were no more sorted in the ascending order as before. This PR sorts the resources (paths) and the schemas.

### CODE REVIEW

File: src/OpenApi/Model/Paths.php [hunk 1]
```diff
@@ -29,6 +29,8 @@ public function getPath(string $path): ?PathItem
 
     public function getPaths(): array
     {
-        return $this->paths ?? [];
+        ksort($this->paths);
+
+        return $this->paths;
```
Comment (soyuka) [inline src/OpenApi/Model/Paths.php:34]: This should be done in the OpenApiNormalizer https://github.com/api-platform/core/blob/main/src/OpenApi/Serializer/OpenApiNormalizer.php#L59 not here.

Comment (stiffroy) [reply src/OpenApi/Model/Paths.php:34]: @soyuka Thanks. I was thinking to do it in the Normalizer first, then thought of doing it in the origin of the data. Should I change it then or did you take over the task?

Comment (soyuka) [reply src/OpenApi/Model/Paths.php:34]: thanks @stiffroy  I took the liberty of hanlding this myself!

**Comment by soyuka** [comment]
superseeded by https://github.com/api-platform/core/pull/4013

**Pull request closed without merging**